app/src/main/java/org/mozilla/reference/browser/components/Toolbar.kt

```diff
@@ -71,12 +71,16 @@ class Toolbar(
             SimpleBrowserMenuItem("Share") {
                 val url = sessionManager.selectedSession?.url ?: ""
                 context.share(url)
+            }.apply {
+                visible = { sessionManager.selectedSession != null }
```

Comment:
🤔  Should we also include a way to make visible again, these menus when we have a new session?

Reply:
The menu builder will get invoked every time the menu button is pressed. :)

Reply:
👍